lib/support/chromePerflogParser.js

```diff
@@ -470,7 +470,11 @@ module.exports = {
           break;
         case 'Network.resourceChangedPriority': {
             let entry = entries.find((entry) => entry._requestId === params.requestId);
-            entry.request._priority = message.params.newPriority;
+
+            if(entry && entry.request) {
```

Comment:
An entry always has a request set, so you can use the same pattern as https://github.com/sitespeedio/browsertime/blob/master/lib/support/chromePerflogParser.js#L327 and other places where we check for entry.